docs/openapi_generator/generate.py

```diff
@@ -25,7 +25,7 @@
 
 
 def str_presenter(dumper, data):
-    if data.startswith(f"/{LLAMA_STACK_API_VERSION}") or data.startswith(
+    if data.startswith(f"/{LLAMA_STACK_API_V1}") or data.startswith(f"/{LLAMA_STACK_API_V1ALPHA}") or data.startswith(f"/{LLAMA_STACK_API_V1BETA}") or data.startswith(
```

Comment:
The second two options will never evaluate, as the first will always take precedent

Reply:
true, thanks!